addons/pos_sale/static/src/app/order_management_screen/sale_order_management_screen/sale_order_management_screen.js

```diff
@@ -165,7 +165,7 @@ export class SaleOrderManagementScreen extends Component {
                     order_id: currentPOSOrder,
                 };
 
-                if (line.display_type === "line_note") {
+                if (["line_note", "line_section"].includes(line.display_type)) {
                     if (previousProductLine) {
```

Comment:
```suggestion
                    if (line.display_type === "line_note" && previousProductLine) {
```
This will not handle the sections correctly as, if the line before the section is a product, it will put the section name as the customer note of this line.

Reply:
I updated the code, but I didn't exactly do your suggestion.